data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -297,10 +297,6 @@
 
 CONTROLLED_TIER_DEID_CLEANING_CLASSES = [
     (RtCtPIDtoRID,),
-    (GenerateExtTables,),
-    (COPESurveyVersionTask,
-    ),  # Should run after GenerateExtTables and before CleanMappingExtTables
-    (PopulateSurveyConductExt,),
```

Comment:
Both of these rules reference the survey data set but with different names.  It's better to use the same name so we only have to pass the value once instead of twice.  Please update this as well.